test/change.test.js

```diff
@@ -601,10 +610,90 @@ describe('Change', function() {
           modelName: updateRecord.modelName,
           prev: null, // this is the current local revision
           rev: 'new-rev',
+          tenant: null,
         });
 
         done();
       });
     });
   });
 });
+
+describe('Change with multi tenancy', function() {
```

Comment:
Since we are testing the generic ability of `Change` model to support extra properties, the test name should not refer to "multi tenancy". I am proposing `Change with custom properties`.